api/app.md

```diff
@@ -500,6 +500,41 @@ func main() {
 ```
 {% endcode %}
 
+## GetRoutes
+
+This method gets get all routes, when filterUse equal to true. it will filter the routes registered by the middleware.
```

Comment:
The first part of the sentence should be "This method gets all routes".

Reply:
thanks

Reply:
I adjusted the document